.gitignore

```diff
@@ -3,3 +3,5 @@
 output
 header.html
 tags
+go.mod
+go.sum
```

Comment:
new line..

Reply:
resolved in 028a295df6524cf8884c409ec121a5c32257608a